packages/zoe/test/swingsetTests/zoe/vat-dave.js

```diff
@@ -75,28 +75,10 @@ const build = async (log, zoe, issuers, payments, installations, timer) => {
       const { value: invitationValue } = await E(invitationIssuer).getAmountOf(
         exclInvitation,
       );
-      const { source } = await E(installation).getBundle();
-      // pick some arbitrary code points as a signature.
-      assert(
-        source.includes('asset: give.Asset,'),
-        X`source bundle didn't match at "asset: give.Asset,"`,
-      );
-      assert(
-        source.includes('swap(zcf, firstSeat, matchingSeat)'),
-        X`source bundle didn't match at "swap(zcf, firstSeat, matchingSeat)"`,
-      );
-      assert(
-        source.includes('makeMatchingInvitation'),
-        X`source bundle didn't match at "makeMatchingInvitation"`,
-      );
-      assert(installation === installations.atomicSwap, X`wrong installation`);
-      assert(
-        sameStructure(
-          harden({ Asset: invitationIssuer, Price: bucksIssuer }),
-          issuerKeywordRecord,
-        ),
-        X`issuerKeywordRecord were not as expected`,
-      );
+
+      // TODO
+      // const { sha256 } = await E(installation).getBundle();
+      // assert.is(sha256, '');
```

Comment:
I see this is just in draft, so I'm getting ahead of ourselves here, but let's make sure to announce this in Discord as a breaking change before we merge this PR, and let's make sure the commits are labeled such that there's a bump to the Zoe version when this gets released. We should have the documentation changes ready to go as well before merging this.

Reply:
I’ve included this commit to probe whether it’s a necessary change to get past CI. I’ll advance it to the top of stack and finish the job if it’s necessary. That would include adding a hash and migrating this test, not deleting it.

Switching to Zip changes the structure of the default bundle object regardless, so I’ll also make sure to annotate the breaking change.

Reply:
sounds great!

Reply:
To address this comment, I’ve annotated the commit that switches the default bundle format as a breaking change and prepared a statement for our support channels:

> *BREAKING CHANGE*: We will soon land a change in Agoric SDK that will change the default bundle format. Any program that currently depends on bundles having a `source` string will break after upgrading Agoric SDK. For example, if you have a program that checks the integrity of a Zoe installation by calling `E(installation).getBundle()` then spot-checking the `source` of the bundle, that program will soon break. We will soon provide a reliable alternative in the form of bundle hash integrity checking. Zip Bundles: https://github.com/Agoric/agoric-sdk/pull/3273 Bundle hashing: https://github.com/Agoric/agoric-sdk/issues/3859